staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -585,7 +584,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				handler = restfulGetResource(getter, exporter, reqScope)
 			}
 			handler = metrics.InstrumentRouteFunc(action.Verb, resource, subresource, handler)
-			handler = genericfilters.RestfulWithCompression(handler, a.group.Context)
+			// handler = genericfilters.RestfulWithCompression(handler, a.group.Context)
```

Comment:
Why not remove entirely...? including `staging/src/k8s.io/apiserver/pkg/server/filters/compression.go` this is the only reference to `RestfulWithCompression` in the codebase.